### PR TITLE
Refactor state / Fixed Slider / Added Stepper and Vue layers with state

### PR DESCRIPTION
**Pull request by waridrox**
## Description

- Refactored state according to https://vuex.vuejs.org/guide/modules.html
- Downgrading to Vue 2 broke the slider for point config. This has been fixed
- Added stepper for Point Configs, RothC and GCBM runs
- Vue layers with input params for latitude and longitude with state saving in vuex store.
- Multiple file upload component at GCBM dashboard for uploading various files.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Additional Context (Please include any Screenshots/gifs if relevant)

...

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings
- [ ] I have tagged the reviewers in a comment below in case my pull request is ready for a review
- [ ] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

**Comment by aornugent** [comment]
Thanks @waridrox - this is huge! It'll take me some time to work throguh it, but at a first glance it looks great. Well done! 🚀

File: flint.ui/src/main.js [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
-
 import VueCompositionAPI from "@vue/composition-api"
 import store from "./store";
-
-// router setup
 import routes from "./routes/routes";
-// import VueCircleSlider from "vue-circle-slider";
 import Toast from "vue-toastification";
+import VueLayers from 'vuelayers';
+import VueSlider from 'vue-slider-component'
```
Comment (aornugent) [inline flint.ui/src/main.js:12]: @shubhamkarande13 indicated on #33 that you were using Vuetify for the slider component. If that is not the case we can remove Vuetify from from his PR.

**Review by aornugent — COMMENTED** [review]
I love the `store/modules` approach. Can you please document this design choice in #9?

Please remove `flint_` from the module names, i.e. just 'point.js', 'rothc.js'.

**Review by aornugent — COMMENTED** [review]
I'm excited to see the stepper (#5) in action! Could you please add a screenshot of the GCBM dashboard?

File: flint.ui/src/main.js (hunk 1 above)
Comment (shubhamkarande13) [reply flint.ui/src/main.js:12]: I did not try `vue-slider-component` as I was not sure about Vue 2 support. 😅
I will remove Vuetify from my PR.

Comment (shubhamkarande13) [reply flint.ui/src/main.js:12]: Done!

**Comment by waridrox** [comment]
There are unstaged files on my local machine, which are not getting adding to the staging area whatsoever, I think there has been a divergence at some point, plus the added creation of duplicate files with `.vue~commit_message` confirms this...Will try to fix the inconsistencies caused due to merging this in a new PR.

**Comment by waridrox** [comment]
Nonetheless, here are the updated changes as a preview: 

https://user-images.githubusercontent.com/58583793/128644670-629ef764-3768-414c-a647-ff48710de597.mov

**Pull request merged**